package.json

```diff
@@ -22,7 +22,7 @@
     "@types/node": "^22.7.9",
     "@typescript-eslint/eslint-plugin": "^7.8.0",
     "@typescript-eslint/parser": "^7.8.0",
-    "@typescript-eslint/rule-tester": "7.5.0",
+    "@typescript-eslint/rule-tester": "^8.13.0",
```

Comment:
```suggestion
    "@typescript-eslint/rule-tester": "8.13.0",
```

Reply:
And you want to re-run yarn and get an updated lock file after this

Reply:
Removed the update of rule-tester. It sucked.